Pod/Classes/WPImageExporter.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface WPImageExporter : NSObject
+
++ (BOOL)writeImage:(UIImage *)image metadata:(NSDictionary *)metadata toURL:(NSURL *)url;
+
++ (NSURL *)URLForTemporaryFileWithFileExtension:(NSString *)fileExtension;
```

Comment:
Nitpick, but we could probably shorten this a tiny bit: `temporaryFileURLWithExtension:`